PageIndex.md

```diff
@@ -92,10 +92,11 @@ a binary search in `min_values` and `max_values`. For unordered columns, a
 reader can find matching pages by sequentially reading `min_values` and
 `max_values`.
 
-For range scans this approach can be extended to return ranges of rows, page
+For range scans, this approach can be extended to return ranges of rows, page
 indices, and page offsets to scan in each column. The reader can then
 initialize a scanner for each column and fast forward them to the start row of
 the scan.
 
 
 
+
```

Comment:
Nit: we could trim the trailing whitespace.

Reply:
removed

Reply:
Using the Apache Parquet documentation to check if Grammarly works properly? ;-)